### ISSUE TITLE
Enable Windows CI 

### ISSUE THREAD

**Issue by kittaakos** [theia-ide/theia#147]
Once #135 is resolved, figure out a way how to run CI on Windows.

Depends on #135.

**Comment by hexa00** [comment]
I found this: https://www.appveyor.com/

It is used by electron see: https://github.com/electron/electron/pull/565

**Comment by kittaakos** [comment]
I have started to work on this task. (Just to avoid doing the same job twice.)

**Comment by akosyakov** [comment]
It would be nice to have the badge for windows builds on the main page + enable builds for PRs.

**Comment by kittaakos** [comment]
PR: #380
